releases/net472/dotnet472-known-issues.md

```diff
@@ -14,3 +14,4 @@ This document lists the known issues that you may experience after you install t
 - [613745 - BCL - Single-name references are removed by the SDK when targeting 4.7.2](https://github.com/Microsoft/dotnet/blob/master/releases/net472/KnownIssues/613745%20-%20Single-name%20references%20are%20removed%20by%20the%20SDK%20when%20targeting%204.7.2.md)
 - [663475 - Setup - Applications that rely on TLS 1.2 strong encryption experience connectivity failures after a Windows upgrade](https://support.microsoft.com/help/4458166)
 - [669153 - Remoting calls made within a transaction can lead to Transaction.Current incorrectly returning null](https://github.com/Microsoft/dotnet/blob/master/releases/net472/KnownIssues/669153%20-%20Remoting%20calls%20made%20within%20a%20transaction%20can%20lead%20to%20Transaction.Current%20incorrectly%20returning%20null.md)
+- [695400 - Setup skips installing payload on Windows 7 to Windows 10 migration](https://github.com/Microsoft/dotnet/blob/master/releases/net472/KnownIssues/695400%20-%20Setup%20skips%20installing%20payload%20on%20Windows%207%20to%20Windows%2010%20migration%20.md)
```

Comment:
Suggestion: "4.7.2 setup fails to install the product after migrating OS from Windows 7 or Windows Server 2008 R2"

Reply:
@rakeshsinghranchi originally has "4.7.2 setup" in the title, but I asked him to remove it, @vivmishra, since all of the issues here are for 4.7.2 -- it seems redundant. What do you think?

Reply:
Sounds good